web-app/js/portal/utils/ObservableUtils.js

```diff
@@ -0,0 +1,16 @@
+Ext.namespace('Portal.utils');
+
+Portal.utils.ObservableUtils = {
+    FUNCTIONS_TO_FORWARD: ['on', 'un', 'fireEvent'],
```

Comment:
I think it'd be good if those could be controlled by the user of this interface. i.e. one could specify what events to forward. Makes this module more reusable down the track.